routes/web.php

```diff
@@ -14,29 +14,33 @@
 |
 */
 
-Route::get('/', 'MatchdayController@index');
-Route::get('/alletermine', 'MatchdayController@index');
+Route::get('/', 'MatchdayController@index')->middleware('auth');
+Route::get('/alletermine', 'MatchdayController@index')->middleware('auth');
 
 Route::get('/termine/bearbeiten', function () {
     return view('matchdays.edit');
-});
+})->middleware('auth');
 
-Route::get('/termine/erstellen', 'MatchdayController@create');
+Route::get('/termine/erstellen', 'MatchdayController@create')->middleware('auth');
 
-Route::get('/allespieler', 'PlayerController@index');
+Route::get('/allespieler', 'PlayerController@index')->middleware('auth');
 
 Route::get('/spieler', function () {
     return view('players.show');
-});
+})->middleware('auth');
 
 Route::get('/spieler/bearbeiten', function () {
     return view('players.edit');
-});
+})->middleware('auth');
 
-Route::get('/spieler/erstellen', 'PlayerController@create');
+Route::get('/spieler/erstellen', 'PlayerController@create')->middleware('auth');
```

Comment:
You could also use a middleware group. This might be more readable. https://laravel.com/docs/7.x/middleware#middleware-groups

```
Route::middleware('auth')->group(function () {
    Route::get('/allespieler', 'PlayerController@index');
    Route::get('/spieler/erstellen', 'PlayerController@create');
    // ...
});
```